Portal/View/Portal.swift

```diff
@@ -12,6 +12,10 @@ public struct Font: AutoEquatable {
     
     let name: String
     
+    public init(name: String) {
```

Comment:
Isn't this `init` generated automatically?

Reply:
yep, but it's internal by default and now the only form to create a Font is with this init.